### ISSUE TITLE
Circular design for user cards

### ISSUE THREAD

**Issue by demerzel3** [demerzel3/wall-of-shame#4]
I would like to change the design of the users cards from the current rectangular one (avatar, PR dots, name) to a circular design. See this as a step toward a new layout that adapts to the available screen real estate.

**Why**
The current design has issues with long names (e.g. francescomalatesta), that I think will just worsen with dynamically-sized cards.

**How**
Here's my proposal:
![schermata 2016-08-20 alle 20 52 54](https://cloud.githubusercontent.com/assets/442835/17833252/1b2727b6-6718-11e6-9bfe-9c27b55482d6.png)
 
In this design all the information is inscribed in a circle, it works better with long names and, being a fixed-size square, is simpler to work with layout-wise.

What do you think? @daBarbe @maxcanna

**Comment by maxcanna** [comment]
I like it! :+1: Regarding those cosmetic improvements I was thinking about using real user names instead of nicknames. As an option maybe. Do you think it's possible or will it require extra work/calls?

**Comment by demerzel3** [comment]
Usernames is all we can get directly from the API calls we currently do, in order to get real names I think we need to dig into the repository log. I will look into it and let you know.

**Comment by maxcanna** [comment]
Repo log? I think that a call to the user endpoint should suffice. That would be a lot of requests though. That's what I was worried about

**Comment by demerzel3** [comment]
You're right, a call to the user endpoint is enough to get the name. 
We already do one call per each PR, I don't think an additional one per user will hurt us 😄

**Comment by maxcanna** [comment]
![](http://i2.kym-cdn.com/entries/icons/original/000/018/147/shialabeouf.jpg)

**Comment by daBarbe** [comment]
I like it! What will happen with long names and a lot of PRs??

**Comment by demerzel3** [comment]
I don't think anyone of us will have more than.. say 5-6 **active** PRs at any given time, we could also decide to limit the number of displayed PRs sorting by date of last update.

About names I don't now, we could use an ellipsis after a certain length.

**Comment by maxcanna** [comment]
> I don't think anyone of us will have more than.. say 5-6 active PRs at any given time

You don't have to think locally. We've to handle everything. I'd go for chronological sorting but showing before the **broken** PRs

**Comment by demerzel3** [comment]
> I'd go for chronological sorting but showing before the broken PRs

I think strict chronological is better. Think abandoned PRs: they eventually become broken as times goes on (e.g. conflicts with the base branch, new rules, etc), if you sort by broken first you will have to display them forever.

**Comment by maxcanna** [comment]
I agree with you. My intention was to show the broken ones before to avoid the situation in which you have the green outline but all red dots. That would be strange but perhaps it's just a problem of mine
